sechub-analyzer-cli/src/test/java/com/daimler/sechub/analyzer/core/ProcessorTest.java

```diff
@@ -0,0 +1,273 @@
+package com.daimler.sechub.analyzer.core;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.daimler.analyzer.model.Marker;
+import com.daimler.analyzer.model.MarkerPair;
+import com.daimler.analyzer.model.MarkerType;
+import com.daimler.analyzer.model.AnalyzerResult;
+
+/**
+ * Integration Tests
+ * 
+ * Tests the Processor and FileAnalyzer classes
+ */
+public class ProcessorTest {
+    
+    final String path = "src/test/resources/";
+    private Processor processor = null;
```

Comment:
avoid setting fields to null - is always done in java automatically and reduces readability